lib/DocumentInspectorFactory.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\DocumentManager;
+
+/**
+ * Document inspector factory for the default DocumentInspector.
+ */
+class DocumentInspectorFactory implements DocumentInspectorFactoryInterface
```

Comment:
Didn't we say we remove this class? I thought you've added the comment somewhere, but I can't find it anymore... Shouldn't that have been merged with the contexts (because I've already seen the contexts in here)?